File: FileCreate/Golang/BashFile.go
package main

import (
	"fmt"
	"log"
	"os"
	"strings"
)

func main() {
	FileCreate()
}

func FileCreate() {
	code := `#!/bin/bash

echo ""
`
	fileName := os.Args[1]
	isextention := strings.HasSuffix(fileName, ".sh")
	if isextention {
		file, err := os.Create(fileName)

		if err != nil {
			log.Fatal(err)
		}

		defer file.Close()

		_, err2 := file.WriteString(code)

		if err2 != nil {
			log.Fatal(err2)
		}

		fmt.Println("Done!")

	} else {
		fmt.Println("Bash File should have extension of .sh")
	}

}

File: PY/BashFile.py
import sys

code = """#!/bin/bash
    
"""

def main():
    arg1 = sys.argv[1]
    print(arg1)

    if arg1.endswith(".sh"):
        f = open(arg1, "w")
        f.write(code)
        f.close()
    else:
        print("Bash file should have .sh extension!")

if __name__ == "__main__":
    main()

